workflows/Create_Incident_in_ServiceNow_from_Slack/workflow.spec.yaml

```
@@ -0,0 +1,29 @@
+extension: workflow
+products: ["insightconnect"]
+name: Create_Incident_in_ServiceNow_from_Slack
+title: "Create Incident in ServiceNow from Slack"
+description: This workflow allows you to automatically create an incident in ServiceNow directly from Slack. 
```

Since this is the first thing someone sees and it is already of type workflow, would recommend:
```suggestion
description: Automatically create an incident in ServiceNow directly from Slack. 
```